zulip/integrations/rss/rss-bot

```diff
@@ -241,7 +248,7 @@ for feed_url in feed_urls:
             # entries in reverse chronological order.
             break
 
-        feed_name = data.feed.title or feed_url  # type: str
+        feed_name = opts.topic
```

Comment:
While this change sets the topic, this doesn't preserve the existing behavior, when a fixed topic is not specified.

Reply:
should be addressed via: https://github.com/zulip/python-zulip-api/pull/747/commits/4de095269de917feb2409c54bd5171bcbcd8f79c